halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/DeploymentEnvironmentValidator.java

```diff
@@ -112,4 +114,14 @@ private void validateDistributedDeployment(ConfigProblemSetBuilder p, Deployment
               + " is not in a provider that supports distributed installation of Spinnaker yet");
     }
   }
+
+  private void validateLivenessProbeConfig(ConfigProblemSetBuilder p, DeploymentEnvironment n) {
```

Comment:
I was thinking this might be confusing to people if they expect to be able to enable the probe in one command and add the delay seconds in a followup command, but hopefully if we document this well people will just copy/paste a functioning command from the docs (which would enable and set initialDelaySeconds in a single command)

Reply:
Also we can remove this error / set a reasonable default initialDelaySeconds in the future when service startup times are more consistent

Reply:
I think this is reasonable as is!  And agree that adding a default eventually would be good (once we are in a place where we can pick one).